cumulus/parachains/runtimes/collectives/collectives-polkadot/src/lib.rs

```diff
@@ -410,6 +410,7 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ControllerOrigin = EitherOfDiverse<EnsureRoot<AccountId>, Fellows>;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
+	// TODO: This should have the price set
```

Comment:
When will this be done?

Reply:
I'll do it right now. It's the same as on the other parachains. The PR has to get audited anyway and this parachain not having this config item for now doesn't block audit work